src/appleseed/renderer/modeling/project/projectfileupdater.cpp

```diff
@@ -1585,6 +1585,47 @@ namespace
             }
         }
     };
+
+    //
+    // Update from revision 21 to revision 22.
+    //
+    
+    class UpdateFromRevision_21
+      : public Updater
+    {
+      public:
+        explicit UpdateFromRevision_21(Project& project)
+          : Updater(project, 21)
+        {
+        }
+
+        void update() override
+        {
+            if (Scene* scene = m_project.get_scene())
+            {
+                for (each<CameraContainer> i = cameras; i; ++i)
```

Comment:
`scene.cameras()`, as is the code doesn't compile.

By the way, we now switched to C++11 so feel free to use range-based for loops:

    for (Camera& camera : scene->cameras())

Reply:
Sorry for PR before check :(
Then shall I change other loops to range-based?